### PR TITLE
[12.0][MIG] sale_packaging

### PR DESCRIPTION
**Pull request by rousseldenis**
Depends on :

* [x] https://github.com/OCA/product-attribute/pull/486
* [ ] Complete coverage
* [x] Remove entry in oca_dependencies.txt

### CODE REVIEW

File: sale_packaging/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+{
+    "name": "Sale Packaging",
+    "version": "12.0.1.0.0",
+    "author": 'ACSONE SA/NV, '
+              'Odoo Community Association (OCA)',
+    "category": "Warehouse",
+    "development_status": "Stable/Production",
+    "maintainers": ["rousseldenis"],
+    "website": "http://github.com/OCA/sale-workflow",
```
Comment (tbaden) [inline sale_packaging/__manifest__.py:12]: would be nice to use tls: https://github.com/OCA/sale-workflow

File: sale_packaging/views/sale_order_line_views.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="sale_order_view_form_inherit_sale_packaging" model="ir.ui.view">
```
Comment (tbaden) [inline sale_packaging/views/sale_order_line_views.xml:3]: use of original_id would be nice (view_order_form), but is optional

File: oca_dependencies.txt [hunk 3]
```diff
@@ -1 +1,4 @@
 server-tools
+
+#TODO: REMOVE ME
+packaging_uom https://github.com/acsone/product-attribute 12.0-add-packaging_uom-dro
```
Comment (tbaden) [inline oca_dependencies.txt]: Todo as mentioned.

**Review by tbaden — COMMENTED** [review]
thanks for the work 👍 
some comments beforehand.
waiting for dependencies.

**Comment by emagdalenaC2i** [comment]
@rousseldenis Is this the migration of sale_packaging_price from v10 or a new module?

**Comment by rousseldenis** [comment]
> @rousseldenis Is this the migration of sale_packaging_price from v10 or a new module?

It's the migration from 10 on the same module but moved from stock-logistics-warehouse as :

https://github.com/OCA/stock-logistics-warehouse/issues/604

**Comment by rafaelbn** [comment]
Please @rousseldenis could you rebase?

I want to have this tested and merged next week as latest date!

**Comment by rafaelbn** [comment]
Ping @chienandalu

**Comment by rousseldenis** [comment]
@chienandalu This one is rebased and ready to be tested.

**Comment by Kev-Roche** [comment]
hi @tbaden, @chienandalu , is it ready to be merged ?
Thank you

**Comment by chienandalu** [comment]
@Kev-Roche I'd say @rousseldenis could say better. I didn't review as far as I remember, but you can do it if you're interested

**Comment by andres-pcg** [comment]
Hello, any update on this one? @chienandalu @rafaelbn

**Comment by chienandalu** [comment]
Sorry @andrp92 but we're not into this module anymore. Maybe some other author can take a look

**Pull request closed without merging**